apps/admin-portal/src/api/identity-provider.ts

```diff
@@ -466,6 +467,41 @@ export const updateOutboundProvisioningConnector = (
         });
 };
 
+/**
+ * Update JIT provisioning configs of a specified IDP.
+ *
+ * @param idpId ID of the Identity Provider.
+ * @param configs JIT provisioning configs.
+ * @return {Promise<any>} A promise containing the response.
+ */
+export const updateJITProvisioningConfigs = (
+    idpId: string,
+    configs: JITProvisioningResponseInterface
+): Promise<any> => {
+
+    const requestConfig = {
+        data: configs,
+        headers: {
+            "Accept": "application/json",
+            "Access-Control-Allow-Origin": store.getState().config.deployment.clientHost,
+            "Content-Type": "application/json"
+        },
+        method: HttpMethods.PUT,
+        url: store.getState().config.endpoints.identityProviders + "/" + idpId +
+            "/provisioning/jit"
+    };
+
+    return httpClient.request(requestConfig)
+        .then((response) => {
+            if (response.status !== 200) {
+                return Promise.reject(new Error("Failed to update identity provider: " + idpId));
+            }
+            return Promise.resolve(response.data as IdentityProviderInterface);
+        }).catch((error) => {
+            return Promise.reject(error);
```

Comment:
Shall we throw `IdentityAppsApiException` here.

Reply:
Fixed in c0e366925209b7d80078ad6edc9077bb13ca9b5b